CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## [5.47.4](https://github.com/plivo/plivo-dotnet/tree/v5.47.4) (2024-09-03)
+**Feature - Adding filter support for List Application API**
+- Added new filer param `appName` in list application API
```

Comment:
we have added keep alive right insteas of this app name ?

Reply:
Updated